src/main/clojure/clara/rules/compiler.clj

```diff
@@ -1263,105 +1439,70 @@
    id :- sc/Int
    is-root :- sc/Bool
    children :- [sc/Any]
-   compile-expr ; Function to do the evaluation.
+   id-key->expr :- {(schema/tuple sc/Int sc/Keyword) IFn}
```

Comment:
Similar as I mentioned above in this ns, perhaps calling it `expr-fn-lookup` is more meaningful. In this local scope, it is less important that where I mentioned above though.